### PR TITLE
Use `selector-observer` in `show-associated-branch-prs-on-fork`

### PR DESCRIPTION
**Pull request by yakov116**
1. LINKED ISSUES:
#2671

2. TEST URLS:
   https://github.com/yakov116/refined-github/branches

### CODE REVIEW

**Review by fregante — APPROVED** [review]
Looks right

File: source/features/show-associated-branch-prs-on-fork.tsx [hunk 1]
```diff
@@ -113,10 +115,6 @@ void features.add({
 	exclude: [
 		() => !pageDetect.isForkedRepo()
 	],
-	init() {
-		observeElement([
-			'[data-target="branch-filter-controller.results"]', // Pre "Repository refresh" layout
-			'[data-target="branch-filter.result"]'
-		].join(), init);
-	}
+	waitForDomReady: false,
```
Comment (fregante) [inline source/features/show-associated-branch-prs-on-fork.tsx:118]: Can be dropped because it can't be avoided
```suggestion
```

**Comment by fregante** [comment]
Great! It follows https://github.com/sindresorhus/refined-github/issues/3084#issuecomment-683345700

Also the diff is super clear when hiding the whitespace changes

File: source/features/show-associated-branch-prs-on-fork.tsx (hunk 1 above)
Comment (yakov116) [reply source/features/show-associated-branch-prs-on-fork.tsx]: We would gain that the request is sent before dom ready

**Pull request merged**